flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -741,6 +741,9 @@ public boolean isSysoutLoggingEnabled() {
 	}
 
 	public GlobalJobParameters getGlobalJobParameters() {
```

Comment:
Nice to mark it as `@NonNull` now.

Reply:
I think the community agreed that all return values which are not annotated with `@Nullable` are meant to be implicitly `@NonNull`. Hence theres should be no need for it.